### ISSUE TITLE
Keep the same file base name for cropped file

### ISSUE THREAD

**Issue by javaone199** [hnvn/flutter_image_cropper#484]
Keep the same file name for croppedFile. 

For example, /path/employee/john-20240301.jpg. Its cropped file name would be like
"image-cropper-12345678.jpg".

The cropped file base name should be the same: john-20240301. The file extension can be changed.
When the file (photo) is uploaded, the file name is meaningful on server side.

File rename and copy is workaround, but it is not necessary if cropper keeps the same base name.

**Comment by Gaurav-CareMonitor** [comment]
Yes this will be really helpful, @javaone199  can you please create a PR for the same
